08_deploy_bmo.sh

```diff
@@ -0,0 +1,14 @@
+#!/usr/bin/bash
+
+# Get the latest bits for baremetal-operator
+export BMOPATH="$GOPATH/src/github.com/metalkube/baremetal-operator"
```

Comment:
GOPATH isn't necessarily defined at this point, so we need to either source ocp_install_env.sh or do `eval "$(go env)"` here